docs/template-cluster-cr.md

```diff
@@ -103,4 +105,43 @@ status:
   provider:
     network:
       cidr: ""
+---
+apiVersion: infrastructure.giantswarm.io/v1alpha2
+kind: G8sControlPlane
+metadata:
+  creationTimestamp: null
+  labels:
+    cluster-operator.giantswarm.io/version: 2.1.10
+    giantswarm.io/cluster: o4omf
+    giantswarm.io/control-plane: osss7
+    giantswarm.io/organization: giantswarm
+    release.giantswarm.io/version: 11.2.1
+  name: osss7
+  namespace: default
+spec:
+  infrastructureRef:
+    apiVersion: infrastructure.giantswarm.io/v1alpha2
+    kind: AWSControlPlane
+    name: osss7
+    namespace: default
+  replicas: 1
+status: {}
```

Comment:
The status is like this here but I see that the status is rendered differently for the other CRs it seems. IMO this is the level of detail we should pay attention to. Should we track that and get it more right?

Reply:
My thinking is that actually the CRs we generate here don't need any status. The YAML created is supposed to be submitted via `kubectl apply`. So `status: {}` should be good enough.